develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -0,0 +1,381 @@
+        "boolValue": false
+      }
+    },
+    {
+      "key": "double_item",
+      "value": {
+        "doubleValue": "value"
+      }
+    }
+  ]
+}
+```
+
+`time_unix_nano`
+
+: **String, optional**. The time the log was created in Unix nanoseconds. If not set, the SDK should set this to the current time.
+
+### `log` Envelope Item Payload
+
+The `log` envelope item payload is a JSON object that represents a log.
+
+TODO: Document `log` Envelope Item Payload once finalized.
+
+### Log Severity Level
+
+The log severity level is a string that represents the severity of the log. The Sentry's default log severity level maps exactly to [OpenTelemetry's Severity text field](https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-severitytext) on the OpenTelemetry Logs Spec..
+
+- `trace`
+- `debug`
+- `info`
+- `warn`
+- `error`
+- `fatal`
+
+### Log Severity Number
+
+The log severity number is an integer that represents the severity of the log. The Sentry's default log severity number maps exactly to [OpenTelemetry's Severity number field](https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-severitynumber) on the OpenTelemetry Logs Spec.
+
+| SeverityNumber range | Range name | Meaning                                                                                 |
+| -------------------- | ---------- | --------------------------------------------------------------------------------------- |
+| 1-4                  | Trace      | A fine-grained debugging event. Typically disabled in default configurations.           |
+| 5-8                  | Debug      | A debugging event.                                                                      |
+| 9-12                 | Info       | An informational event. Indicates that an event happened.                               |
+| 13-16                | Warn       | A warning event. Not an error but is likely more important than an informational event. |
+| 17-20                | Error      | An error event. Something went wrong.                                                   |
+| 21-24                | Fatal      | A fatal error such as application or system crash.                                      |
+
+Default SDK public API should set the lowest severity number for a given severity level. For example, `warn` severity level logs collected by the SDK API should use the severity number `13`.
+
+## Public API
+
+API wise the SDKs are required to expose logging methods which are to be defined in a `logger` module or namespace. The SDKs should also include some initialization options to configure the behavior of logs in the SDK.
+
+### Initialization Options
+
+The SDKs need to expose the following two configuration options:
+
+- `enableSentryLogs`/`enable_sentry_logs`: A boolean flag to control if log envelopes will be generated and sent to Sentry via the Sentry SDK's Logging APIs. If this flag is set to `false`, the SDK should not send logs to Sentry. Defaults to `false`.
```

Comment:
Bikeshed: I would just call it `enable_logs`. It is a property given to the `sentry_sdk.init()` command so I think there is no need to put `sentry` in the config option. None of the other options we have has sentry in it. It is also called `enable_tracing` and not `enable_sentry_tracing` because the context of the init() funciton makes it clear that it is sentry related tracing (or logs).

Reply:
Done with 2d2abe8372c4d3806c777d3a274585a6f40ea9ae